e2e/all-charts-spec.ts

```diff
@@ -1,25 +1,26 @@
-const using = require('jasmine-data-provider');
-const helper = require('./helpers/helper');
-
-const CommonChartSidebar = require('./pages/CommonChartSidebar-page');
-
-const BubbleChartPage = require('./pages/BubbleChart-page');
-const MapChartPage = require('./pages/MapChart-page');
-const LineChartPage = require('./pages/LineChart-page');
-const MountainChartPage = require('./pages/MountainChart-page');
-const RankingsChartPage = require('./pages/RankingsChart-page');
-
-let commonChartSidebar = new CommonChartSidebar();
-
-let DATA_PROVIDER = {
-  'Bubbles Chart': {chart: new BubbleChartPage()},
-  'Map Chart': {chart: new MapChartPage()},
-  'Mountains Chart': {chart: new MountainChartPage()},
-  'Line Chart': {chart: new LineChartPage()},
-  'Rankings Chart': {chart: new RankingsChartPage()}
+import { browser, protractor } from 'protractor';
+
+import { Helper } from './helpers/helper';
+import { CommonChartSidebar } from './pages/CommonChartSidebar-page';
+import { BubbleChart } from './pages/BubbleChart-page';
+import { RankingsChart } from './pages/RankingsChart-page';
+import { LineChart } from './pages/LineChart-page';
+import { MapChart } from './pages/MapChart-page';
+import { MountainChart } from './pages/MountainChart-page';
+
+import using = require('jasmine-data-provider');
+
+const commonChartSidebar = new CommonChartSidebar();
```

Comment:
type of commonChartSidebar is CommonChartSidebar. Will looks like:
 +const commonChartSidebar:  CommonChartSidebar = new CommonChartSidebar();

Reply:
same with another variables.